index.js

```diff
@@ -1,21 +1,66 @@
 import '@ethersproject/shims';
 import './shim';
 import '@walletconnect/react-native-compat';
-import {AppRegistry} from 'react-native';
+import {AppRegistry, Alert} from 'react-native';
 import 'react-native-get-random-values';
 import Root from './src/Root';
 import React from 'react';
 import './i18n';
+import {
+  setJSExceptionHandler,
+  setNativeExceptionHandler,
+} from 'react-native-exception-handler';
 import {name as appName} from './app.json';
 import getStore from './src/store';
 import {Provider} from 'react-redux';
 import {PersistGate} from 'redux-persist/integration/react';
 import 'react-native-url-polyfill/auto'; // https://github.com/facebook/react-native/issues/23922#issuecomment-648096619
 import {enableFreeze} from 'react-native-screens';
+import {Analytics} from './src/store/analytics/analytics.effects';
+import {APP_VERSION} from './src/constants/config';
+import {GIT_COMMIT_HASH} from '@env';
+import {LogActions} from './src/store/log';
 enableFreeze(true);
 
 export const {store, persistor} = getStore();
 
+const errorHandler = (e, isFatal) => {
+  if (isFatal) {
+    store.dispatch(
+      Analytics.track('BitPay App - Crashed App', {
+        build: GIT_COMMIT_HASH,
+        version: APP_VERSION,
+      }),
+    );
+    const errStr = e instanceof Error ? e.message : JSON.stringify(e);
+    LogActions.persistLog(LogActions.error(errStr));
```

Comment:
This needs to be dispatched as an store action

```
store.dispatch(LogActions...)
```

Reply:
Yes, fixed. Thanks!